modules/nw-secondary-ext-gw-configure.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-external-gateway.adoc
+
+:_content-type: PROCEDURE
+[id="nw-secondary-ext-gw-configure_{context}"]
+= Configure a secondary external gateway
+
+You can configure a secondary external gateway for a namespace in your cluster.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a YAML file that contains an `AdminPolicyBasedExternalRoute` object.
+. To create an admin policy based external route, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f <file>.yaml
+----
++
+--
+where:
+
+`<file>`:: Specifies the name of the YAML file that you created in the previous step.
+--
++
+.Example output
+[source,text]
+----
+adminpolicybasedexternalroute.k8s.ovn.org/default-route-policy created
+----
+
+. To confirm that the admin policy based external route was created, enter the following command:
++
+[source,terminal]
+----
+$ oc describe apbexternalroute <name> | tail -n 6
+----
++
+--
+where:
+
+`<name>`:: Specifies the name of the `AdminPolicyBasedExternalRoute` object.
+--
++
+.Example output
+[source,text]
+----
+Status:
+  Last Transition Time:  2023-04-24T15:09:01Z
+  Messages:
+  Configured external gateway IPs: 172.18.0.8
+  Status:  Success
+Events:  <none>
+----
+
+.Verification
+
+If you created an `AdminPolicyBasedExternalRoute` object that selects a host-network pod IP address as the secondary external gateway, you can confirm that the next hop is correct for a pod with the following steps:
+
+. To get the IP address of the pod, enter the following command:
++
+[source,terminal]
+----
+oc get pods/<pod_name> -n <namespace> -o wide
+----
++
+--
+where:
+
+`<pod_name>`:: Specifies the name of the pod.
+`<namespace>`:: Specifies the namespace of the pod.
+--
+
+. Confirm that the IP address from the previous step is available as an external gateway.
+
+.. To find the the OVN-Kubernetes control plane pod that manages the next hop for the pod, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app=ovnkube-master -n openshift-ovn-kubernetes | grep <node_name>
+----
++
+--
+where:
+
+`<node_name>`:: Specifies the name of the node from the `NODE` column that the pod from the previous step is running on.
+--
+
+.. To confirm that the OVN-Kubernetes node pod includes the correct next hop, enter the following command:
++
+[source,terminal]
+----
+$ oc exec -t <pod_name> -n openshift-ovn-kubernetes -- ovn-nbctl \
+  list logical_router_static_route | grep 10.244.2.6 -A 6 -B 4
```

Comment:
@jab-rh   Jose suggested "ovn-nbctl lr-route-list GR_ovn-work", but also notice we're it's executed, not in the same place the current command suggests.   

This part gets us to the correct POD name inside worker-0-1:  POD=$(kubectl get pod -n openshift-ovn-kubernetes --field-selector spec.nodeName=worker-0-1 -o jsonpath='{range .items[*]}{.[metadata.name](http://metadata.name/)}{"\n"}{end}' | grep ovnkube-node-)

This part starts a bash session inside the correct nbdb container:  kubectl exec -ti $POD -n openshift-ovn-kubernetes -c nbdb -- bash

Once inside we'd execute:  ovn-nbctl lr-route-list GR_ovn-work